README.md

```diff
@@ -439,9 +441,9 @@ archive { '/tmp/staging/master.zip':
   (default: false)
 * `extract_path`: target folder path to extract archive.
 * `extract_command`: custom extraction command ('tar xvf example.tar.gz'), also
-* `temp_dir`: specify an alternative temporary directory to use for file downloads, if unset the OS default is used
-  support sprintf format ('tar xvf %s') which will be processed with the
-  filename: sprintf('tar xvf %s', filename)
+* `temp_dir`: specify an alternative temporary directory to use for file downloads,
+  if unset the OS default is used support sprintf format ('tar xvf %s') which will
```

Comment:
The structure of the sentence seems a bit confusing? Is there a `.` missing between `used` and `support`?

Reply:
Good catch. This got messed up 4 years ago. 
* https://github.com/voxpupuli/puppet-archive/blob/master/lib/puppet/type/archive.rb#L92-L103